src/scene/scenes/AssemblyScene.tsx

```diff
@@ -20,9 +23,12 @@ const AssemblyScene: SceneComponent = () => (
         <meshBasicMaterial transparent opacity={0} />
       </Box>
     </ExplodeTrigger>
-    <Orbiter position={[0, 2, 0]} scale={orbiterScale} />
-    <AssembleTestSceneName position={[0, 7, 2]} scale={sceneNameScale} />
+    <Orbiter position={[0, 6, -2]} scale={orbiterScale} />
+    <AssembleTestSceneName position={[0, 12, -1]} scale={sceneNameScale} />
     <AssembleAnimation />
+    <ModelOutliner color={0xffffff}>
```

Comment:
`SceneManager` already wraps all of the in a `ModelOutliner` so you should not need it here.